akka-docs/src/main/paradox/actors.md

```diff
@@ -880,6 +880,8 @@ timeout there must have been an idle period beforehand as configured via this me
 Once set, the receive timeout stays in effect (i.e. continues firing repeatedly after inactivity
 periods). Pass in `Duration.Undefined` to switch off this feature.
```

Comment:
I think this should go as well (even if that also works), better to document one way of doing it.
```suggestion
periods).
```

Reply:
sure